src/postamble_minimal.js

```diff
@@ -68,6 +68,12 @@ var imports = {
 #endif
 };
 
+// In non-fastcomp non-asm.js builds, grab wasm exports to outer scope
+// for emscripten_get_exported_function() to be able to access them.
+#if (LibraryManager.has('library_exports.js')) && (WASM || WASM_BACKEND)
```

Comment:
The condition here and below seem a little unfortunate - it starts to clutter the otherwise nice and simple minimal runtime's postamble code. Can we avoid them by using `asm` directly and not add this new `wasmExports`?

(To put it another way, surely the exports are needed anyhow?)

Reply:
Ok, simplified the code to reuse the `asm` variable name.